cps/main.c

```diff
@@ -1119,6 +1119,12 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		cps_conf->log_ratelimit_interval_ms,
 		cps_conf->log_ratelimit_burst);
 
+	/* Take the packets needed in the KNI into account as well. */
+	net_conf->front.total_pkt_burst += 2 * cps_conf->front_max_pkt_burst;
+	if (net_conf->back_iface_enabled)
```

Comment:
If `run_cps()` fails after here and it's called again, `net_conf->front.total_pkt_burst` and `net_conf->back.total_pkt_burst` will have inflated values. The solution is to stack a failure label to correct it before returning during a failure. Using local variables to hold the increments helps to avoid duplicating the formulas in the code.

This comment applies to all other functional blocks.

Reply:
Please apply local variables to all functional blocks to avoid duplicate formulas.